tests/Package/PacakgeTest.php

```diff
@@ -24,12 +24,12 @@ class PacakgeTest extends TestCase
     protected function setUp(): void
```

Comment:
Just noticed this filename is typod: `Pacakge` => `Package`

Reply:
Oh good catch! Thx, fixed